_templates/searchbox.html

```diff
@@ -0,0 +1,16 @@
+<!-- Empty override file take the place of searchbox.html, which generates search box. -->
+<intl>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
This would have to use `sphinx-design` for which we already load the CSS. I think they do all kinds of things, including grids, which I already use, and combo boxes for sure too, however it aims at rest. But I guess it can be taken from a Sphinx generated file too. 

External CSS kills page load times and cannot be done. Also I want flags to be used, although that begs the question, which one English is going to use. I found this one: https://flagpack.xyz/ from that we can have a flag there, that on hover loads the other language images or something like that. For Google search ranking, page load times are critical, and therefore we cannot load 20 flags really, so one is already a lot.

Not sure how to treat partial translations and untranslated pages in this switching world. Maybe there is a default content for untranslated pages, or maybe we do not make it switchable unless it's complete.

What is your take on that?

Reply:
https://flagpack.xyz/ is a good idea.

I don't think users care too much about complete translation. They just want to get the information they want quickly. We just need to state the general progress of translation in 'intl/index.rst'.

Reply:
How to use `sphinx-design` in `_templates/searchbox.html`, I don't know. But I agree.

Reply:
Checkout this: https://staging.nuitka.net/doc/welcome.html the debian "flag" is a dropdown like one we might consider, but I am worried about the performance impact of doing that with flags. Might want to use only flag for current language and words for others. I demonstrated in the rest how to do images where normally only text is allowed. That is a similar trick as used on the download page.

In my mind, much like social buttons, I would do it in the "footer" and use CSS absolute positioning to place it at the top of the page. I think it is common practice to have it on the right side. So near social buttons, there would be the flag on translated content or always.

If you want to play with it, checkout the `my_theme.css` and how it places the social buttons.

I did not manage to find the PNG files yet, that are mentioned in here: https://flagpack.xyz/docs/design

I installed npm react variant of their package, but it's only SVG, but I understand for small sizes (and we will be using a relatively small one), SVG will be worse.

Reply:
Ok, this being PNGs is only in my head, seems they have just 3 sets of SVGs designed for the sizes.

Reply:
I have no idea about it. I only know a little about Web.